cli/project/dockerfiles.go

```diff
@@ -181,6 +184,42 @@ func getInstallTarantoolLayers(ctx *context.Ctx) (string, error) {
 	return installTarantoolLayers, nil
 }
 
+func getVersionForTarantoolInstaller(version string) string {
+	parts := strings.SplitN(version, ".", 3)
+	major, _ := strconv.Atoi(parts[0])
+	minor, _ := strconv.Atoi(parts[1])
+
+	if major > 2 || (major >= 2 && minor > 8) {
+		return fmt.Sprintf("%d", major)
+	}
+
+	return fmt.Sprintf("%d.%d", major, minor)
+}
+
+func getOpenSourceInstallerLink(version string) string {
```

Comment:
The function returns a repository type, not a link.

Reply:
Fixed